build fails due to incorrect location of libleveldb.a

This is on Debian Jessie/Sid, with Linux kernel 4.0.0 on amd64.

````
g++ -o ldb ldb.cc ./lib/*.cc -I./deps/leveldb/include -std=gnu++11 -lpthread ./deps/leveldb/libleveldb.a -lsnappy ./linenoise.o
g++: error: ./deps/leveldb/libleveldb.a: No such file or directory
Makefile:28: recipe for target 'ldb' failed
make: *** [ldb] Error 1
````

I can see that the libleveldb.a file is not in the `deps/leveldb` directory, but one level down, in the `out-static` directory:
 
````
~/src/ldb$ ls deps/leveldb/
AUTHORS          build_detect_platform*  db/   helpers/  issues/  Makefile  out-shared/  port/      table/  util/
build_config.mk  CONTRIBUTING.md         doc/  include/  LICENSE  NEWS      out-static/  README.md  TODO
~/src/ldb$ ls deps/leveldb/out-static/libleveldb.a
deps/leveldb/out-static/libleveldb.a
````

--- a/Makefile
+++ b/Makefile
@@ -4,7 +4,7 @@ LIBPATH ?= ./lib
 SRC = ldb.cc $(LIBPATH)/*.cc
 PREFIX ?= /usr/local
 LEVELDBPATH ?= ./deps/leveldb
-LIBLEVELDB ?= $(LEVELDBPATH)/libleveldb.a
+LIBLEVELDB ?= $(LEVELDBPATH)/out-static/libleveldb.a
 CXXFLAGS += -I$(LEVELDBPATH)/include -std=gnu++11
 
 DEPPATH ?= ./deps

